mypy/stubgen.py

```diff
@@ -561,9 +560,14 @@ def _get_func_return(self, o: FuncDef, ctx: FunctionContext) -> str | None:
             if has_return_statement(o):
                 return_name = self.add_name("_typeshed.Incomplete")
             return f"{generator_name}[{yield_name}, {send_name}, {return_name}]"
+        if o.is_property:
+            return None
+
+        if is_trivial_body(o.body) and not self.legacy:
+            return self.add_name("_typeshed.Incomplete")
```

Comment:
what is this

Reply:
```py
def foo(): ...
```
has a 'trivial' body, and will generate the stub:
```py
def foo() -> Untyped: ...
```

Reply:
actually, what the spruce was I thinking!